### PR TITLE
Removes incorrect PY_SCIP_CALL wrapper

### PR DESCRIPTION
**Pull request by Opt-Mucca**
This fixes the issue raised in #591

### CODE REVIEW

**Comment by CGraczyk** [comment]
Please update the changelog before merge, thx!

File: src/pyscipopt/scip.pyx [hunk 1]
```diff
@@ -1930,8 +1930,8 @@ cdef class Model:
         return SCIPgetRowNumIntCols(self._scip, row.scip_row)
 
     def rowGetNNonz(self, Row row):
-        """Gets number of non-zero etnries in the row"""
-        return PY_SCIP_CALL(SCIProwGetNNonz(row.scip_row))
+        """Gets number of non-zero entries in the row"""
+        return SCIProwGetNNonz(row.scip_row)
```
Comment (fserra) [inline src/pyscipopt/scip.pyx:1934]: I would say this function shouldn't exist. First, it doesn't use the scip pointer so it shouldn't be in class Model. Second, this function already is implemented in the class Row.

So maybe the proper fix is to just delete the function :)

Comment (Opt-Mucca) [reply src/pyscipopt/scip.pyx]: Fair point, was wondering if it was needed because of the duplicate. Nice to know about the SCIP pointer rule.

**Comment by Opt-Mucca** [comment]
Added a line to the CHANGELOG and deleted the function following Felipe's advice

**Pull request merged**